nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/src/main/java/org/apache/nifi/processors/standard/InvokeHTTP.java

```diff
@@ -558,24 +553,19 @@
     public static final Set<Relationship> RELATIONSHIPS = Collections.unmodifiableSet(new HashSet<>(Arrays.asList(
             REL_SUCCESS_REQ, REL_RESPONSE, REL_RETRY, REL_NO_RETRY, REL_FAILURE)));
 
-    private volatile Set<String> dynamicPropertyNames = new HashSet<>();
+    // RFC 2616 Date Time Formatter with hard-coded GMT Zone
+    private static final DateTimeFormatter RFC_2616_DATE_TIME = DateTimeFormatter.ofPattern("EEE, dd MMM yyyy HH:mm:ss 'GMT'");
 
-    /**
-     * Pattern used to compute RFC 2616 Dates (#sec3.3.1). This format is used by the HTTP Date header and is optionally sent by the processor. This date is effectively an RFC 822/1123 date
-     * string, but HTTP requires it to be in GMT (preferring the literal 'GMT' string).
-     */
-    private static final String RFC_1123 = "EEE, dd MMM yyyy HH:mm:ss 'GMT'";
-    private static final DateTimeFormatter DATE_FORMAT = DateTimeFormat.forPattern(RFC_1123).withLocale(Locale.US).withZoneUTC();
+    // Multiple Header Delimiter
+    private static final String MULTIPLE_HEADER_DELIMITER = ", ";
 
-    private final AtomicReference<OkHttpClient> okHttpClientAtomicReference = new AtomicReference<>();
+    private volatile Set<String> dynamicPropertyNames = new HashSet<>();
 
-    @Override
-    protected void init(ProcessorInitializationContext context) {
-        excludedHeaders.put("Trusted Hostname", "HTTP request header '{}' excluded. " +
-                "Update processor to use the SSLContextService instead. " +
-                "See the Access Policies section in the System Administrator's Guide.");
+    private volatile Pattern regexAttributesToSend = null;
 
-    }
```

Comment:
Understanding that this was intended to catch previous uses, sending an HTTP Header named `Trusted Hostname` will no impact on the received HTTP server unless that server was explicitly configured to look for that property.  Since it is a non-standard HTTP Header, and it is up to the receiving system to do something with it, performing the check in this processor seems unnecessary.